core/kazoo_number_manager/src/knm_numbers.erl

```diff
@@ -701,10 +701,10 @@ check_creditably(_Services, _Quotes, Amount) when Amount =< 0 ->
 check_creditably(Services, _Quotes, Amount) ->
     Options = #{amount => kz_currency:dollars_to_units(Amount)},
     case kz_services:is_good_standing(Services, Options) of
-        'true' -> 'ok';
-        'false' ->
+        {'true', _} -> 'ok';
```

Comment:
The spec for this function is wrong, takes a `boolean() | integer()` in the last argument, no?